packages/storybook/stories/Adapters/Adapters.stories.mdx

```diff
@@ -186,3 +186,17 @@ async function createAzureCommunicationCallAdapter(args: AzureCommunicationChatA
 | **`credential`**  | [`CommunicationTokenCredential`](https://docs.microsoft.com/en-us/javascript/api/@azure/communication-common/azurecommunicationtokencredential) |
 | **`endpoint`**    | `string`                                                                                                                                        |
 | **`threadId`**    | `string`                                                                                                                                        |
+
+### createAzureCommunicationChatAdapterFromClient
+
+```ts
+async function createAzureCommunicationChatAdapterFromClient(
+  chatClient: StatefulChatClient,
+  chatThreadClient: ChatThreadClient,
+  options?: {
```

Comment:
```suggestion
  options?: ChatAdapterOptions
```
We have a type here we can call out.  
And below we can have a chart that is showcases the credential argument inside.